src/discussions/discussions-home/DiscussionsHome.jsx

```diff
@@ -89,9 +89,9 @@ export default function DiscussionsHome() {
           path={[Routes.POSTS.PATH, Routes.TOPICS.CATEGORY]}
           component={LegacyBreadcrumbMenu}
         />
-        <div className="d-flex flex-row">
+        <div className="d-flex flex-row vh-100">
```

Comment:
I think `vh-100` is usually used when the container should be the same height as the view port, which is not the case here. We want the container to take the rest of the space in the flexbox wrapper, so `flex-grow-1` can (I want to say 'should', but I'm not 100% confident) be used here.

Reply:
Also, you should add `overflow-hidden` on `main` and this container, so the children can be scrollable. [Ref](https://stackoverflow.com/a/35609992/15935523)

Reply:
I did not know this! Thanks! I've replaced the vh-100 with overflow hidden